app/docs/enterprise/0.31-x/plugins/request-transformer.md

```diff
@@ -178,11 +187,22 @@ Append multiple headers and remove a body parameter:
 | p1=v1&p2=v1 | p2=v1
 | p2=v1 | p2=v1
 
+Add multiple headers and add a querystring parameter if not already set:
+
+```
+$ curl -X POST http://localhost:8001/apis/mockbin/plugins \
+  --data "name=request-transformer-advanced" \
+  --data "config.add.headers=h1:v1,h2:v1" \
+  --data "config.add.querystring=q1:v2,q2:v1" \
```

Comment:
the example says add _multiple_ headers and _a_ (meaning a single one as opposition to multiple?) querystring parameter, but we are adding two querystrings here

Reply:
Good eye!